src/main/java/fr/openmc/core/features/hdv/HDVListing.java

```diff
@@ -0,0 +1,22 @@
+package fr.openmc.core.features.hdv;
+
+import lombok.Getter;
+import org.bukkit.inventory.ItemStack;
+
+import java.time.LocalDateTime;
+import java.util.UUID;
+
+@Getter
+public class HDVListing {
+    private final UUID seller;
+    private final ItemStack item;
+    private final double price;
+    private final LocalDateTime createdAt;
+
+    public HDVListing(UUID seller, ItemStack item, double price) {
+        this.seller = seller;
+        this.item = item.clone();
+        this.price = price;
+        this.createdAt = LocalDateTime.now();
+    }
+}
```

Comment:
Un nom comme `HDVSlot` serait plus explicite. Actuellement, le nom `HDVListing` prête à confusion, car il suggère une liste alors que cette classe ne contient que les informations d’un seul « slot » de l’HDV.

```suggestion
public class HDVSlot {
    private final UUID seller;
    private final ItemStack item;
    private final double price;
    private final LocalDateTime createdAt;

    public HDVSlot (UUID seller, ItemStack item, double price) {
        this.seller = seller;
        this.item = item.clone();
        this.price = price;
        this.createdAt = LocalDateTime.now();
    }
}
```